cmd/cgr-tester/cgr-tester.go

```diff
@@ -289,4 +301,18 @@ func main() {
 	} else {
 		log.Printf("Elapsed: %s resulted: %f req/s.", duration, float64(*runs)/duration.Seconds())
 	}
+
+	var wg sync.WaitGroup
```

Comment:
I do not see a way to differentiate between getCost and session Simulates, maybe I am missing something?